### PR TITLE
sql file added for databases week3 homework

### PR DESCRIPTION
**Pull request by rismita87**
Databases week3 homework

### CODE REVIEW

File: databases/week3/main.sql [hunk 1]
```diff
@@ -0,0 +1,254 @@
+USE dbweek3HW;
```
Comment (mateusms01) [inline databases/week3/main.sql:3]: would also include the create database statement in this file.

File: databases/week3/main.sql [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+  meal
+WHERE
+  meal.id = 2;
+--4.Update a meal with any id, fx 1.
+  --Update any attribute fx the title or multiple attributes
+UPDATE
+  meal
+SET
+  max_reservations = "20",
+  price = "95"
```
Comment (mateusms01) [inline databases/week3/main.sql]: well done updating multiples

File: databases/week3/main.sql [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+WHERE
+  review.id = "3";
+--Additional queries
+  --1.Get meals that has a price smaller than a specific price fx 90
+SELECT
+  *
+FROM
+  meal
+WHERE
+  meal.price > 80;
```
Comment (mateusms01) [inline databases/week3/main.sql]: technically the query asks for smaller (<) than a specific price.

File: databases/week3/main.sql [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+--2.Get meals that still has available reservations
+SELECT
+  meal.id "meal id",
+  meal.title "meal title",
+  reservation.number_of_guests "reservation number"
+FROM
+  meal
+  INNER JOIN reservation ON meal.id = reservation.meal_id
+WHERE
+  meal.max_reservations > reservation.number_of_guests;
```
Comment (mateusms01) [inline databases/week3/main.sql]: this query returns reservations that have number_of_quests bigger than max_reservations, which is not the number of reservations made to that specific meal. 

The correct answer would be achieved by counting how many distinct reservation_id's are grouped by meal_id and than filtering for max_reservation > count of distinct reservation id's

File: databases/week3/main.sql [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+WHERE
+  title LIKE '%chicken%';
+--4.Get meals that has been created between two dates
+SELECT
+  *
+FROM
+  meal
+WHERE
+  created_date BETWEEN '2022-03-23'
+  AND '2022-03-24';
```
Comment (mateusms01) [inline databases/week3/main.sql:231]: for readability and clarity condition should be in one liner.

**Review by mateusms01 — CHANGES_REQUESTED** [review]
Really good work! Minor comments left

**Comment by rismita87** [comment]
Thanks a lot Mateus for your feedback. I'll do the corrections accordingly
:)

regards
Rismita

File: databases/week3/main.sql (hunk 1 above)
Comment (rismita87) [reply databases/week3/main.sql:3]: ok,done.

File: databases/week3/main.sql (hunk 3 above)
Comment (rismita87) [reply databases/week3/main.sql]: done.

File: databases/week3/main.sql (hunk 4 above)
Comment (rismita87) [reply databases/week3/main.sql]: thanks, I misunderstood. I changed it accordingly.

File: databases/week3/main.sql (hunk 5 above)
Comment (rismita87) [reply databases/week3/main.sql:231]: ok.

**Pull request merged**